haystack/components/converters/docx.py

```diff
@@ -119,6 +121,7 @@ def __init__(self, table_format: Union[str, DOCXTableFormat] = DOCXTableFormat.C
             If False, only the file name is stored.
         """
         docx_import.check()
+        lxml_import.check()
```

Comment:
```suggestion
```
if we incorporate the `lxml` import in the docx lazy import block, we can remove this line.